nerdlets/container-explorer/container-explorer.js

```diff
@@ -125,7 +125,7 @@ export default class ContainerExplorer extends React.Component {
 
     const facets = await getCardinality({
       eventType: 'ProcessSample',
-      accountId: account.id,
+      accountIds: account.id,
```

Comment:
unless `getCardinality` was updated, I don't think we want this change. (`getCardinality` is a local function)